db/seeds.rb

```diff
@@ -17,20 +32,14 @@
 end
 
 [
-  ["A Clockwork Orange", "0393312836", "db/sample_data/a-clockwork-orange.jpg", ["Dystopian Fiction"]],
-  ["Crime and Punishment", "0679734503", "db/sample_data/crime-and-punishment.jpg", ["Psychological Fiction", "Russian Literature"]],
-  ["Transcending CSS", "0321410971", "db/sample_data/transcending-css.jpg"],
-  ["The Very Hungry Caterpillar", "0399226907", "db/sample_data/the-very-hungry-caterpillar.jpg", ["Children's Literature"]]
-].each do |name, isbn, cover, genre_names = []|
+  ["A Clockwork Orange", "0393312836", "db/sample_data/a-clockwork-orange.jpg", ["Anthony Burgess"], ["Dystopian Fiction"]],
+  ["Crime and Punishment", "0679734503", "db/sample_data/crime-and-punishment.jpg", ["Fyodor Dostoevsky"], ["Psychological Fiction", "Russian Literature"]],
+  ["Transcending CSS", "0321410971", "db/sample_data/transcending-css.jpg", ["Andy Clarke", "Molly E. Holzschlag"],[]],
+  ["The Very Hungry Caterpillar", "0399226907", "db/sample_data/the-very-hungry-caterpillar.jpg", ["Eric Carle"], ["Children's Literature"]]
+].each do |name, isbn, cover, author_names = [], genre_names = []|
   book = Book.find_or_create_by(name: name, isbn: isbn)
+  matching_authors = authors.select { |author| author_names.include?(author.name) }
+  book.update(authors: matching_authors)
   matching_genres = genres.select { |genre| genre_names.include?(genre.name) }
   book.update(genres: matching_genres)
-  unless book.cover.attached?
-    book.cover.attach(
-      content_type: "image/jpg",
-      filename: cover.split("/").last,
-      io: File.open(Rails.root.join(cover)),
-    )
-  end
 end
```

Comment:
Out of curiosity, why did you remove the covers?

Reply:
A rebase error! Returned.